### PR TITLE
Adjust the Missouri deductions metadata to align with model standards

### LINKED ISSUE #4327: Adjust the Missouri deductions metadata to align with model standards
Currently the descriptions and labels need to be changed:
- Descriptions in full sentences 
etc.

### PR DESCRIPTION
**Pull request by Kevintowne**
Fixes #4327

### CODE REVIEW

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: remove all changes related to the expenses

File: policyengine_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax/ignored_credits.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax/ignored_credits.yaml:8]:
```suggestion
  label: Missouri ignored credits in federal income tax deduction
  period: year
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax/ignored_credits.yaml:8]: add period everywhere, where missing

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/mo/tax/income/credits/property_tax/public_assistance_types.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/credits/property_tax/public_assistance_types.yaml:10]:
```suggestion
  label: Missouri property tax credit gross income public assistance sources
```

File: policyengine_us/parameters/gov/states/mo/tax/income/credits/property_tax/rent_property_tax_limit.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/credits/property_tax/rent_property_tax_limit.yaml:7]:
```suggestion
  label: Missouri property tax credit limit for property renters
```

File: policyengine_us/parameters/gov/states/mo/tax/income/credits/wftc/eitc_match.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/credits/wftc/eitc_match.yaml:6]:
```suggestion
  label: Missouri EITC match
```

File: policyengine_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax/ignored_credits.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax/ignored_credits.yaml:7]:
```suggestion
  unit: list
```

File: policyengine_us/variables/household/expense/utilities/broadband_cost.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/household/expense/utilities/broadband_cost.py:1]: revert all utility expense related variables - wrong PR

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/mo/tax/income/deductions/mo_max_private_pension.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/deductions/mo_max_private_pension.yaml:8]:
```suggestion
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Comment by MaxGhenis** [comment]
Merging this. Thank you. But let's hold off on further cleanup PRs until merging all possible commits from #4210. I did much of this sort of work there so it would duplicate work and cause merge conflicts.

**Pull request merged**
